deepview_profile/analysis/session.py

```diff
@@ -204,7 +231,8 @@ def energy_compute(self) -> pm.EnergyResponse:
             logger.error(message)
             resp.analysis_error.error_message = message
         except Exception:
```

Comment:
This should have `Exception as ex` as well

Reply:
done